.editorconfig

```diff
@@ -45,7 +45,7 @@ insert_final_newline = true
 [*.cs]
 
 #### File Header Template ####
-file_header_template = Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file to you under the MIT license.\nSee the LICENSE file in the project root for more information.
+file_header_template = Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file under the MIT license.
```

Comment:
We should be consistent with .NET I agree, which appears to removed the third line and is now:

```
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
```

But we should not edit this text otherwise, we're not lawyers, we can't make those calls. We need to use the standard text. If .NET updates or we get guidance from the Foundation otherwise should be the only reason to change this. FYI @Sergio0694

Reply:
Makes perfect sense to me 👍
Maybe we could open an issue to ask about this in the .NET Foundation repo?

Reply:
I've asked in the Foundation's discord maintainers channel but will let you know if I see any updated guidance later. In general, I've been asking for more documented guidance for things like this.